server/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
What's this file? Probably shouldn't be here.

Reply:
No idea! removed